impeller/compiler/compiler_test.cc

```diff
@@ -160,6 +160,24 @@ bool CompilerTest::CanCompileAndReflect(const char* fixture_name,
     }
   }
 
+  auto mapping =
```

Comment:
Oh, my bad. I wasn't clear. The code here should output to the right file given the name of the fixture. Probably need to add a `ReflectionDepfileName()`. See how the other files are written out above. Also see [here](https://github.com/flutter/engine/blob/main/impeller/compiler/impellerc_main.cc#L207). Then the code in https://github.com/flutter/engine/blob/main/impeller/compiler/compiler_unittests.cc can pass a fixture with a %20 in the name, and inspect the depfile output.